trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/noise_geometric.py

```diff
@@ -150,4 +150,57 @@ def noise_geometric(noise_segment,analyses,config):
     noise_segment.theta = theta
     noise_segment.phi   = phi
 
-    return (dist,theta,phi)
\ No newline at end of file
+    return (dist,theta,phi)
+
+
```

Comment:
white space

Reply:
I'm going to correct this and delete this pull request per our discussion this morning